### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by ghost**
### Screenshot
Screenshot_16

### CODE REVIEW

File: components/CommandSearch.tsx
Comment (omjogani) [inline components/CommandSearch.tsx:1]: Please move `filteredItems`, `componentsItems`, `faqsItems` in to separate file. so static data can be reside at separate place and won't looks code messy...

File: components/FAQS.tsx
Comment (omjogani) [inline components/FAQS.tsx:1]: @Utkarsh4517 Please check FAQS Questions once... and react with 👍 to this comment.

File: components/Features.tsx
Comment (omjogani) [inline components/Features.tsx:1]: @Utkarsh4517 Please review these features once... and react with 👍 to this comment.

**Review by omjogani — CHANGES_REQUESTED** [review]
Little changes & Other stuff looks rock solid 🔥

**Comment by Utkarsh4517** [comment]
@omjogani everything lgtm.
awesome work @n-georges

File: components/CommandSearch.tsx
Comment (omjogani) [reply components/CommandSearch.tsx:1]: Waiting for this one! @n-georges

**Review by omjogani — APPROVED** [review]
LGTM! Thanks @n-georges

**Pull request merged**